ci/templatetags/seed.py

```diff
@@ -2,9 +2,21 @@
 
 import seed_control_interface
 
+import re
+
 register = template.Library()
 
 
 @register.simple_tag
 def current_version():
     return seed_control_interface.__version__
+
+
+@register.filter
+def unslug(value):
+    return re.sub('[^0-9a-zA-Z]+', ' ', value).strip()
```

Comment:
Is there a reason to use this instead of the simpler `r'\W+'`? I think with a normal string this is equivalent, but in python 3 if this is a unicode string, then `\W` covers us for unicode as well.